adyenv6core/resources/adyenv6core-items.xml

```diff
@@ -540,5 +540,91 @@
                 </attribute>
             </attributes>
         </itemtype>
+
+        <itemtype code="adyenAuditTable" generate="true" autocreate="true">
+            <deployment table="adyenAuditTable" typecode="24244"/>
```

Comment:
why are we using 24244 code if previous was 22936? Wouldn't it be better to use 22937?